src/main/java/org/ow2/proactive/addons/email/EmailSender.java

```diff
@@ -211,6 +252,23 @@ protected void configurePlainTextMessage(MimeMessage message) throws MessagingEx
         message.setText(body);
     }
 
+    protected void configurePlainTextMessageWithAttachment(MimeMessage message) throws MessagingException {
+        configurePlainTextMessage(message);
+
+        MimeBodyPart messageBodyPart = new MimeBodyPart();
+
+        Multipart multipart = new MimeMultipart();
+
+        String file = fileToAttache;
+        String name = fileName;
```

Comment:
please check that both are not null and throw an understandable error message otherwise.